templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  <li>
+    <nav>
+      {% if user.is_authenticated %}
+        <li>User: {{ user.username }}</li>
+        <li><a href="{% url 'logout' %}">Logout</a></li>
+        {% else %}
+        <li><a href="{% url 'login' %}">Login</a></li>
+      {% endif %}
+    <br>
+    </nav>
```

Comment:
The `<nav>` element should not be placed inside a `<li>` element. Consider restructuring the HTML to ensure semantic correctness, such as placing the `<nav>` outside of the `<li>` or using `<div>` or `<span>` for styling purposes.